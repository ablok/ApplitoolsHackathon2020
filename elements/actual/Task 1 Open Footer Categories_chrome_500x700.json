[
  {
    "//h3[text()='Contacts']": [
      {
        "visibility": true,
        "location": {
          "x": 0,
          "y": 2277.796875
        },
        "size": {
          "width": 485,
          "height": 46
        },
        "css": {
          "display": "block",
          "backgroundColor": "rgba(0, 0, 0, 0)",
          "backgroundImage": "none",
          "textDecoration": "none solid rgb(255, 255, 255)",
          "color": "rgb(255, 255, 255)",
          "font": "500 13px / 15.6px Roboto, Arial, Helvetica, sans-serif"
        },
        "text": "Contacts"
      }
    ]
  },
  {
    "#collapse_3": [
      {
        "visibility": true,
        "location": {
          "x": 15,
          "y": 2338.796875
        },
        "size": {
          "width": 455,
          "height": 78
        },
        "css": {
          "display": "block",
          "backgroundColor": "rgba(0, 0, 0, 0)",
          "backgroundImage": "none",
          "textDecoration": "none solid rgb(68, 68, 68)",
          "color": "rgb(68, 68, 68)",
          "font": "14px / 21px Roboto, Arial, Helvetica, sans-serif"
        }
      }
    ]
  },
  {
    "//h3[text()='Keep in touch']": [
      {
        "visibility": true,
        "location": {
          "x": 0,
          "y": 2441.796875
        },
        "size": {
          "width": 485,
          "height": 46
        },
        "css": {
          "display": "block",
          "backgroundColor": "rgba(0, 0, 0, 0)",
          "backgroundImage": "none",
          "textDecoration": "none solid rgb(255, 255, 255)",
          "color": "rgb(255, 255, 255)",
          "font": "500 13px / 15.6px Roboto, Arial, Helvetica, sans-serif"
        },
        "text": "Keep in touch"
      }
    ]
  },
  {
    "#collapse_4": [
      {
        "visibility": true,
        "location": {
          "x": 15,
          "y": 2507.796875
        },
        "size": {
          "width": 455,
          "height": 40
        },
        "css": {
          "display": "block",
          "backgroundColor": "rgba(0, 0, 0, 0)",
          "backgroundImage": "none",
          "textDecoration": "none solid rgb(68, 68, 68)",
          "color": "rgb(68, 68, 68)",
          "font": "14px / 21px Roboto, Arial, Helvetica, sans-serif"
        }
      }
    ]
  },
  {
    "//h3[text()='Quick Links']": [
      {
        "visibility": true,
        "location": {
          "x": 0,
          "y": 2231.796875
        },
        "size": {
          "width": 485,
          "height": 46
        },
        "css": {
          "display": "block",
          "backgroundColor": "rgba(0, 0, 0, 0)",
          "backgroundImage": "none",
          "textDecoration": "none solid rgb(255, 255, 255)",
          "color": "rgb(255, 255, 255)",
          "font": "500 13px / 15.6px Roboto, Arial, Helvetica, sans-serif"
        },
        "text": "Quick Links"
      }
    ]
  },
  {
    "#collapse_1": [
      {
        "visibility": false,
        "location": {
          "x": 0,
          "y": 1787
        },
        "size": {
          "width": 92,
          "height": 182
        },
        "css": {
          "display": "none",
          "backgroundColor": "rgba(0, 0, 0, 0)",
          "backgroundImage": "none",
          "textDecoration": "none solid rgb(68, 68, 68)",
          "color": "rgb(68, 68, 68)",
          "font": "14px / 21px Roboto, Arial, Helvetica, sans-serif"
        }
      }
    ]
  }
]
