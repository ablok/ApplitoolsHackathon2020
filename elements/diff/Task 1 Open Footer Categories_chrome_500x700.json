[
  {
    "//h3[text()='Contacts']": [
      {
        "op": "update",
        "path": [0, "location", "y"],
        "val": 2277.796875,
        "oldVal": 2459.796875
      }
    ]
  },
  {
    "#collapse_3": [
      {
        "op": "update",
        "path": [0, "location", "y"],
        "val": 2338.796875,
        "oldVal": 2520.796875
      }
    ]
  },
  {
    "//h3[text()='Keep in touch']": [
      {
        "op": "update",
        "path": [0, "location", "y"],
        "val": 2441.796875,
        "oldVal": 2623.796875
      }
    ]
  },
  {
    "#collapse_4": [
      {
        "op": "update",
        "path": [0, "location", "y"],
        "val": 2507.796875,
        "oldVal": 2689.796875
      }
    ]
  },
  {
    "#collapse_1": [
      {
        "op": "update",
        "path": [0, "visibility"],
        "val": false,
        "oldVal": true
      },
      {
        "op": "update",
        "path": [0, "location", "x"],
        "val": 0,
        "oldVal": 15
      },
      {
        "op": "update",
        "path": [0, "location", "y"],
        "val": 1787,
        "oldVal": 2292.796875
      },
      {
        "op": "update",
        "path": [0, "size", "width"],
        "val": 92,
        "oldVal": 455
      },
      {
        "op": "update",
        "path": [0, "size", "height"],
        "val": 182,
        "oldVal": 142
      },
      {
        "op": "update",
        "path": [0, "css", "display"],
        "val": "none",
        "oldVal": "block"
      }
    ]
  }
]
