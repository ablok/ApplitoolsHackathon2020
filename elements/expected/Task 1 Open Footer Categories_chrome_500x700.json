[
  {
    "//h3[text()='Contacts']": [
      {
        "visibility": true,
        "location": {
          "x": 0,
          "y": 2458.796875
        },
        "size": {
          "width": 485,
          "height": 46
        },
        "css": {
          "display": "block",
          "backgroundColor": "rgba(0, 0, 0, 0)",
          "backgroundImage": "none",
          "textDecoration": "none solid rgb(255, 255, 255)",
          "color": "rgb(255, 255, 255)",
          "font": "500 13px / 15.6px Roboto, Arial, Helvetica, sans-serif"
        },
        "text": "Contacts"
      }
    ]
  },
  {
    "#collapse_3": [
      {
        "visibility": true,
        "location": {
          "x": 15,
          "y": 2519.796875
        },
        "size": {
          "width": 455,
          "height": 78
        },
        "css": {
          "display": "block",
          "backgroundColor": "rgba(0, 0, 0, 0)",
          "backgroundImage": "none",
          "textDecoration": "none solid rgb(68, 68, 68)",
          "color": "rgb(68, 68, 68)",
          "font": "14px / 21px Roboto, Arial, Helvetica, sans-serif"
        }
      }
    ]
  },
  {
    "//h3[text()='Keep in touch']": [
      {
        "visibility": true,
        "location": {
          "x": 0,
          "y": 2622.796875
        },
        "size": {
          "width": 485,
          "height": 46
        },
        "css": {
          "display": "block",
          "backgroundColor": "rgba(0, 0, 0, 0)",
          "backgroundImage": "none",
          "textDecoration": "none solid rgb(255, 255, 255)",
          "color": "rgb(255, 255, 255)",
          "font": "500 13px / 15.6px Roboto, Arial, Helvetica, sans-serif"
        },
        "text": "Keep in touch"
      }
    ]
  },
  {
    "#collapse_4": [
      {
        "visibility": true,
        "location": {
          "x": 15,
          "y": 2688.796875
        },
        "size": {
          "width": 455,
          "height": 40
        },
        "css": {
          "display": "block",
          "backgroundColor": "rgba(0, 0, 0, 0)",
          "backgroundImage": "none",
          "textDecoration": "none solid rgb(68, 68, 68)",
          "color": "rgb(68, 68, 68)",
          "font": "14px / 21px Roboto, Arial, Helvetica, sans-serif"
        }
      }
    ]
  },
  {
    "//h3[text()='Quick Links']": [
      {
        "visibility": true,
        "location": {
          "x": 0,
          "y": 2230.796875
        },
        "size": {
          "width": 485,
          "height": 46
        },
        "css": {
          "display": "block",
          "backgroundColor": "rgba(0, 0, 0, 0)",
          "backgroundImage": "none",
          "textDecoration": "none solid rgb(255, 255, 255)",
          "color": "rgb(255, 255, 255)",
          "font": "500 13px / 15.6px Roboto, Arial, Helvetica, sans-serif"
        },
        "text": "Quick Links"
      }
    ]
  },
  {
    "#collapse_1": [
      {
        "visibility": true,
        "location": {
          "x": 15,
          "y": 2291.796875
        },
        "size": {
          "width": 455,
          "height": 142
        },
        "css": {
          "display": "block",
          "backgroundColor": "rgba(0, 0, 0, 0)",
          "backgroundImage": "none",
          "textDecoration": "none solid rgb(68, 68, 68)",
          "color": "rgb(68, 68, 68)",
          "font": "14px / 21px Roboto, Arial, Helvetica, sans-serif"
        }
      }
    ]
  }
]
